### ISSUE TITLE
Tempest searches can't find Doctor Who (2005)

### ISSUE THREAD

**Issue by EnderTheDestroyer** [Tempest0580/tempest#81]
Not sure if this is trakt related but I couldn't find Doctor Who (2005) on Tempest today, although the show does show up in searches on The Crew. Just an fyi. Thanks for everything Tempest!

**Comment by Tempest0580** [comment]
> Not sure if this is trakt related but I couldn't find Doctor Who (2005) on Tempest today, although the show does show up in searches on The Crew. Just an fyi. Thanks for everything Tempest!

Just seen this. I've had alot going on. You still having issues with this?

**Comment by EnderTheDestroyer** [comment]
Yeah, it's so frickin weird, it's not just Tempest being affected. A Tempest tv show search doesn't show Doctor Who (2005) at all, which is the entire modern Doctor Who series. but it shows up in The Crew tv show search but the special episodes in Season 0 don't have any playable sources, at least the most recent one anyway. That was the one I was looking to play. I don't use RD, PM, AD btw, just free sources.

It seems to depend on which addon I try, one or two show the 2005 season but a bunch don't, so I couldn't figure out what the heck was going on. I figured with the trakt problems, tmdb rumors, etc. it could be anything. No worries though, this was a little confusing/frustrating to me but no one else seems to be noticing any disappearing Doctor Who's so please put this on the back burner until things settle down, it's literally not a big deal. Thanks for getting back to me, sorry to bug you! Take care Tempest...

**Comment by 0Phant0m** [comment]
Doctor Who (2005)  still showing up for episodes I haven't watched yet. Manually searching for this show via Search under TV Shows still returning Doctor Who (2005), and episodes playable.

While back when I was using Exodus Redux I've observed series that I was yet to be caught up on no longer showing up with the Trakt Progress listing, like Alone for instances.

Another issue since resolvers was switched, TV shows had numerous 1080HD sources to try and now I'm lucky to have one 720HD. When I'm really lucky I'll be seeing three 720HD, not necessarily mean it'll play or play without a hitch. 

Various TV shows I watch has very small number of sources in the list to use like Marvel's Agents of S.H.I.E.L.D. but at least Tempest has some unlike Exodus Redux. However popular shows like this, and one can't find many sources, It's a shame.

**Comment by EnderTheDestroyer** [comment]
I did another TV Shows search in Tempest this morning but the current series dated 2005 didn't show up. I was in the hospital over New Years and missed the Christmas special which is why I went searching for it. Never did find it in Kodi but managed to find the complete episode last night on the web. I did notice that The Crew found the 2005 series and had playable season 12 episodes but I'm caught up on those, it was just the one special episode in season 0 that I was looking for.
